examples/siro_sandbox/README_VR.md

```diff
@@ -0,0 +1,218 @@
+* 103997424_171030444
+* 103997541_171030615
+
+### Dataset Processing
+
+#### Installation
+
+This step requires [Magnum](https://github.com/mosra/magnum) data processing tools. They are not built along with Habitat.
+
+It is recommended that you create a new `conda` environment so that it can be reused in the future without interfering with Habitat.
+
+1. Get the latest `magnum-tools` from [Magnum CI](https://github.com/mosra/magnum-ci/actions/workflows/magnum-tools.yml). Extract to a convenient location.
+2. Create a new `conda` environment:
+```
+conda create --name magnum python=3.10
+```
+3. Navigate to the `site-packages` of your new environment, e.g. `~/anaconda/envs/magnum/lib/python3.10/site-packages/`.
+4. Create a `magnum.pth` file in this directory.
+5. Add the absolute path to `magnum-tools`'s `python` folder to this file, e.g:
+```
+/home/USER/Documents/magnum-tools/linux-x64/python/
+```
+6. The Magnum libraries will now be included upon activating your `magnum` environment. Validate the installation as such:
+   * `conda activate magnum`
+   * `python -c "from magnum import math, meshtools, scenetools, trade"`
+
+#### Usage
+
+Example command:
+
+```
+python get_scene_object_glbs.py \
+--hssd-hab-root-dir path_to/hssd-hab/ \
+--hssd-models-root-dir path_to/hssd-models/ \
+--scenes 102344193 102344280 102817200 103997424_171030444 103997541_171030615
+```
+
+#### Importing Data Into Unity
+
+In Unity, use `Tools/Update Data Folder...`. From the dialog window, copy the path to the generated `data/hitl_simplified/data` in the `External Data Path` field. The resources will be imported into Unity.
+
+After this step, you may deploy the application to your headset. See [instructions](https://github.com/eundersander/siro_hitl_unity_client/blob/main/README.md).
+
+## Connection
+
+Upon launching the server, it will start listening for incoming connections. The client will attempt to connect to the addresses listed in `Android/data/com.meta.siro_hitl_vr_client/files/config.txt`. It rotates between the addresses periodically until a connection is established.
+
+See [troubleshooting notes](#connection-issues) is connection fails.
+
+## Server Controls
+
+* Use the `1-5` keys to change scenes. Scenes are currently hardcoded here.
+* Use `T` to toggle between server-controlled mouse-keyboard and client-controlled VR controls.
+* If the robot gets stuck, hold `O` to temporarily use an oracle policy.
+
+## Troubleshooting
+
+### Deployment to the VR Headset
+
+* Make sure that [developer mode](https://developer.oculus.com/documentation/native/android/mobile-device-setup/) is activated.
+* When connecting your headset to your development computer via USB, a pop-up will ask you to confirm the connection within the VR headset.
+  * If the pop-up doesn't show up, reconnect your USB cable.
+  * You may also have to re-do this after the headset goes into sleep mode.
+* Deployment occasionally fails when the application is already installed.
```

Comment:
An extra tip here: when in doubt about a build failing to push to the headset, I like to manually delete the old build on the headset (can be done from Quest storage menu).